abilities/music/downloads/.gitignore

```diff
@@ -0,0 +1,4 @@
+# This directory exists to contain temporary files downloaded from youtube.
+# The .gitignore prevents git from tracking those binary files
+*
+!.gitignore
```

Comment:
this is just a quick fix before we figure out how to stream the data without persisting any files